cmd/bbolt/main.go

```diff
@@ -1380,10 +1380,10 @@ func (cmd *benchCommand) runReadsSequential(db *bolt.DB, options *BenchOptions,
 
 		for {
 			numReads := int64(0)
+			defer func() { results.AddCompletedOps(numReads) }()
```

Comment:
I debated using a defer or manually adding the call at the end of the loop. I ended up doing it with the defer, as there are some cases in which it would need to be called from inside the if condition if the run had an error and outside, too. So, it feels cleaner. However, I'm unsure if too many defers (one per loop) would create issues with the call stack if there are too many iterations.

Reply:
Adding `defer` inside a for loop isn't a good practice. 

How about this?

```
	err := func() error {
		defer func() { results.AddCompletedOps(numReads) }()

		c := tx.Bucket(benchBucketName).Cursor()
		for k, v := c.First(); k != nil; k, v = c.Next() {
			numReads++
			results.AddCompletedOps(1)
			if v == nil {
				return ErrInvalidValue
			}
		}

		return nil
	}()
	if err != nil {
		return err
	}
```

Reply:
Note that deferred function won't be called until the surrounding function returns. It means the initial motivation of progress notification is broken. My proposed change above can fix it.

Reply:
@ahrtr, I applied the suggestion. PTAL :)